Code/Framework/AzCore/AzCore/Settings/SettingsRegistryConsoleUtils.h

```diff
@@ -57,5 +63,13 @@ namespace AZ::SettingsRegistryConsoleUtils
     //!  Merges the json formatted file <file path> into the settings registry underneath the root anchor ""
     //!  or <anchor json path> if supplied
     [[nodiscard]] ConsoleFunctorHandle RegisterAzConsoleCommands(SettingsRegistryInterface& registry, AZ::IConsole& azConsole);
+
+    //! Registers the following console commands for the SettingsRegistryOriginTracker
+    //! "sr_dump_origin" accepts 0 or more arguments <settings-key path>*
+    //!  Outputs the file origin starting at the supplied settings key path to stdout.
+    //!  The settings keys are recursed to find children settings. Each of those have their file origins output as well
+    //!  If no arguments settings key aarguments are supplied, all settigns file origins are output
```

Comment:
aarguments -> arguments
settigns -> settings

Reply:
To make it similar to update, here is a github suggestion.
```suggestion
    //!  If no arguments settings key arguments are supplied, all settings file origins are output
```